.prettierrc.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
I prefer `JSON` over `YAML`. JSON is closer to JavaScript whereas YAML is closer to CoffeeScript and since we are on the JavaScript path, we should also be on the JSON path.

Reply:
YAML was introduced in 2001.. so has nothing to do with CoffeeScript (2009) and for configuration files it's much more sane format, because among other things it also supports comments and there are no useless symbols like: `{ } , ; " "`

Having said that though.. maybe we should vote

https://vote4code.com/5663284820705280/eslintyaml-vs-eslintjson